gluten-ut/spark33/src/test/scala/org/apache/spark/sql/GlutenDataFrameAggregateSuite.scala

```diff
@@ -200,4 +200,19 @@ class GlutenDataFrameAggregateSuite extends DataFrameAggregateSuite with GlutenS
           _.isInstanceOf[HashAggregateExecBaseTransformer]).isDefined)
     }
   }
+
+  test("mixed supported and unsupported aggregate functions") {
+    val df = spark.sql("SELECT a, collect_set(b), max(b) FROM testData2 group by a")
```

Comment:
Thanks so much for your fix!

Currently, this test can well test the mixed supported/unsupported functions. But when collect_set is supported (actually WIP on my side), the test case will not meet our purpose. Right?
Maybe, we can add some c++ test by designing an agg function with input type that can never match with any velox function's signature. I can help do this.

Reply:
Certainly, it is better to have an end-2-end test like you proposed. But I'm not sure whether we can design one spark test that always meet our test purpose.

Reply:
I agree, all built-in aggregte functions should be offload to native in future. I changed to use a UDAF to make sure it always running on jvm.